.ci-builds/.azure-pipelines.yml

```diff
@@ -703,7 +703,19 @@ jobs:
         mkdir -p icuexportdata/collation/implicithan
         ./bin/genrb -X -s data/coll/ --ucadata data/in/coll/ucadata-implicithan-icu4x.icu -d icuexportdata/collation/implicithan $FILES
         rm icuexportdata/collation/implicithan/*.res
-      displayName: 'Build collation data export files'
+      displayName: 'Build collation data files'
+      env:
+        LD_LIBRARY_PATH: lib
+    - script: |
+        cd icu4c/source
+        mkdir -p icuexportdata/segmenter/dictionary
+        for FILE in `ls data/brkitr/dictionaries`
+        do
+          ./bin/gendict --uchars data/brkitr/dictionaries/$FILE /dev/stdout | \
+          python3 -c 'import sys; data = sys.stdin.buffer.read(); print(f"trie_data = {[data[i + 1] << 8 | data[i] for i in range(64, len(data) - 1, 2)]}")' \
```

Comment:
Should we add copyright headers?

Reply:
Other icuexportdata files don't have them.

Reply:
I thought they all had headers like

```
# Copyright (C) 2021 and later: Unicode, Inc. and others.
# License & terms of use: http://www.unicode.org/copyright.html
#
# file name: AHex
#
# machine-generated by: icuexportdata.cpp
```

Are these headers missing from some of the existing files?